internal/webserver/handler-ui.go

```diff
@@ -320,7 +320,7 @@ func (h *Handler) ServeBookmarkEbook(w http.ResponseWriter, r *http.Request, ps
 	}
 
 	// Check if it has ebook.
-	ebookPath := fp.Join(h.DataDir, "ebook", strID+".epub")
+	ebookPath := fp.Join(h.DataDir, "ebook", strID)
```

Comment:
Changing this will make it not backwards compatible. People that have ebooks stored and upgrade to this change wont be able to download their ebooks, ebook will be downloaded again and the old one (without extension) will be left in the filesystem.

Reply:
We can do something, but it's pretty inconvenient and hard for distributors to build upon. 

Let's move the "ebook exists" logic to one function and let the function:
1. Check the filename without extension
    - If exists, `os.Rename` the file to `id.epub` 
2. Check the `id.epub` extension.

We can remove the first check after some versions.